tools/wptrunner/wptrunner/browsers/edge.py

```diff
@@ -84,6 +85,20 @@ def start(self, **kwargs):
 
     def stop(self, force=False):
         self.server.stop(force=force)
+        # Wait for Edge browser process to exit if driver process is found
+        edge_proc_name = 'MicrosoftEdge.exe'
+        for i in range(0,5):
+            procs = subprocess.check_output(['tasklist', '/fi', 'ImageName eq ' + edge_proc_name])
+            if 'MicrosoftWebDriver.exe' not in procs:
+                # Edge driver process already exited, don't wait for browser process to exit
+                break
+            elif edge_proc_name in procs:
+                time.sleep(0.5)
+            else: break
```

Comment:
The break should be on a new line here (this is the CI failure, flake8 complaining about two statements on one line).

Reply:
I pushed https://github.com/web-platform-tests/wpt/pull/16118/commits/03a0a9fe7e0238f4df6daa2bf9c182d4c2ae69ee